modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/model/listener/UserExpandoColumnModelListener.java

```diff
@@ -201,7 +201,8 @@ else if ((expandoColumn.getType() == ExpandoColumnConstants.DOUBLE) ||
 					 ExpandoColumnConstants.FLOAT_ARRAY)) {
 
 			entityField = new DoubleEntityField(
-				encodedName, locale -> encodedIndexedFieldName);
+				encodedName,
```

Comment:
@joshuacords Please do not hard code logic here, please move the logic to ExpandoBridgeIndexerImpl#encodeFieldName